core/tests/webdriverio_utils/LearnerDashboardPage.js

```diff
@@ -107,6 +107,26 @@ var LearnerDashboardPage = function() {
     expect(titleOfExplorationSummary).toMatch(title);
   };
 
+  this.removeExplorationSummaryTileFromPlaylist = async function(title) {
+    let removeTile = await $(`.e2e-test-exp-summary-tile-title=${title}`);
+    await removeTile.moveTo();
+
+    let removeFromPlayLaterListButton = $(
+      '.e2e-test-remove-from-playlist-btn');
+    await removeFromPlayLaterListButton.waitForDisplayed();
+    await action.click(
+      'Remove from playlist button', removeFromPlayLaterListButton, false);
+    let confirmDeleteInteractionButton = $(
+      '.e2e-test-confirm-delete-interaction');
+    await confirmDeleteInteractionButton.waitForDisplayed();
+    await action.click(
+      'Confirm Delete Interaction button', confirmDeleteInteractionButton);
+    await waitFor.invisibilityOf(
+      confirmDeleteInteractionButton,
+      'Delete Interaction modal takes too long to close');
+    expect(await removeTile.isDisplayed()).toBe(false);
```

Comment:
Please move this expectation to a separate function so that it can be called separately from the test. This makes it easier to see from the code in the `describe` block that we do this assertion. You can follow the pattern we use elsewhere in this file, e.g. `expectTitleOfExplorationSummaryTileToMatch`

Reply:
Done!